pkg/agent/catalog/nodeattestor.go

```diff
@@ -26,7 +26,7 @@ func (repo *nodeAttestorRepository) Constraints() catalog.Constraints {
 }
 
 func (repo *nodeAttestorRepository) Versions() []catalog.Version {
-	return []catalog.Version{nodeAttestorV0{}}
+	return []catalog.Version{nodeAttestorV1{}}
```

Comment:
I think that V0 is missing here, which will prevent plugins that haven't been converted to work.